### PR TITLE
add support for insert tags

### PR DESCRIPTION
**Pull request by simonreitinger**
As discussed in #6, I added the insert tag support.

The default value only gets selected when it is in the possible options.

### CODE REVIEW

**Comment by aschempp** [comment]
I don't think the `$option['default'] = true;` part is necessary, as the widget value already is one of the options?

**Comment by simonreitinger** [comment]
@aschempp that's true! edited in https://github.com/terminal42/contao-countryselect/pull/8/commits/de031b0fc63dc32e554054931d728fe41093a950

File: FormCountrySelectMenu.php [hunk 1]
```diff
@@ -16,10 +16,17 @@ public function addAttributes($arrAttributes)
         parent::addAttributes($arrAttributes);
         $arrOptions = array(array('label' => ($this->placeholder == '' ? '-' : $this->placeholder), 'value' => ''));
         $arrCountries = $this->getCountries();
+        
+        // allow insert tags to be set as default value, e.g. {{user::country}}
+        // see https://github.com/terminal42/contao-countryselect/issues/6
+        if (strpos($this->varValue, '{{') !== false) {
+            $this->varValue = \Contao\Controller::replaceInsertTags($this->value);
```
Comment (aschempp) [inline FormCountrySelectMenu.php]: Is it intentional to use `$this->value` and not `$this->varValue` for the method argument?

Comment (simonreitinger) [reply FormCountrySelectMenu.php]: actually yes! `$this->value` is implemented in `class Widget`.

Comment (aschempp) [reply FormCountrySelectMenu.php]: I know it is, but why read from that field but then write to another one?

Comment (simonreitinger) [reply FormCountrySelectMenu.php]: ok that seems reasonable. Should I use varValue or value only?

Comment (aschempp) [reply FormCountrySelectMenu.php]: I think we should use `$this->varValue` within the class itself. `$this->value` is only for "external" access.

**Comment by Toflar** [comment]
Thank you @simonreitinger

**Comment by simonreitinger** [comment]
No problem!

@aschempp @Toflar should I change `$this->value` to `this->varValue` as this is already merged?

**Comment by Toflar** [comment]
Already merged and released. Doesn't really matter :) Also I've added a new feature to allow selecting the important countries that are then listed on top of all the others :)

**Pull request merged**